# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/UpdateImageHelper.h
# Commit: 96da9d241051
# Full Hash: 96da9d241051d222bdd693bdb519d058a013b59c
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2019-03-23 09:46:24
# Regressor Bug: 1441308
# File Overlap Count: 2
# Description:
#   Bug 1441308 - Core renderroot splitting changes r=kats,sotaro
#   
#   This is a large patch that contains all of the core changes for
#   renderroot splitting.
#   
# ==============================================================================

diff -r 8cefa694f811 -r 96da9d241051 gfx/layers/UpdateImageHelper.h
--- a/gfx/layers/UpdateImageHelper.h	Fri Mar 22 18:28:31 2019 +0000
+++ b/gfx/layers/UpdateImageHelper.h	Fri Mar 22 18:28:42 2019 +0000
@@ -52,7 +52,7 @@
     return target.forget();
   }
 
-  bool UpdateImage() {
+  bool UpdateImage(wr::RenderRoot aRenderRoot) {
     if (!mTexture) {
       return false;
     }
@@ -65,7 +65,8 @@
     RefPtr<TextureWrapperImage> image = new TextureWrapperImage(
         mTexture, gfx::IntRect(gfx::IntPoint(0, 0), mImageSize));
     mImageContainer->SetCurrentImageInTransaction(image);
-    return mImageClient->UpdateImage(mImageContainer, /* unused */ 0);
+    return mImageClient->UpdateImage(mImageContainer, /* unused */ 0,
+                                     Some(aRenderRoot));
   }
 
  private: